openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.interface.ts

```diff
@@ -23,19 +23,13 @@ export enum AssetsOfEntity {
   FOLLOWING = 'FOLLOWING',
 }
 
-export enum AssetsViewType {
```

Comment:
Why we removed this view type?

Reply:
Not using the other ViewType any more. We are following common view in all areas.